flink-kubernetes/src/test/java/org/apache/flink/kubernetes/highavailability/KubernetesHighAvailabilityRecoverFromSavepointITCase.java

```diff
@@ -101,12 +102,14 @@ public void setup() throws Exception {
     @Test
     public void testRecoverFromSavepoint() throws Exception {
         final JobGraph jobGraph = createJobGraph();
-        clusterClient.submitJob(jobGraph).get(TIMEOUT, TimeUnit.MILLISECONDS);
+        clusterClient
+                .submitJob(jobGraph)
+                .get(TestingUtils.infiniteTime().toMilliseconds(), TimeUnit.MILLISECONDS);
```

Comment:
just to be clear: That's just added to comply to with our guidelines for tests to not use timeouts, isn't it? It's not related to the change itself...